test/OfflineTest.php

```diff
@@ -42,6 +44,24 @@ public function setUp()
     }
 
     /**
+     * Enables mocks for ldap_connect(), ldap_bind(), and ldap_set_option().
+     * Not all tests need or are compatible with this, so it is called expliclty
+     * by tests that do.
+     */
+    protected function activateBindableOfflineMocks()
+    {
+        BuiltinFunctionMocks::$ldap_connect_mock->enable();
+        BuiltinFunctionMocks::$ldap_bind_mock->enable();
+        BuiltinFunctionMocks::$ldap_set_option_mock->enable();
+    }
+
+    public function tearDown()
```

Comment:
Don't change the existsing test. If you need special stuff for new tests you should move them to a separate test-class